wiki/en/Unregistered-Servers.md

```diff
@@ -0,0 +1,34 @@
+---
+layout: wiki
+title: "Running an Unregistered Server"
+lang: "en"
+permalink: "/wiki/Unregistered-Servers"
+---
+
+{% include breadcrumb.html root="More" branch1="Server Administration" branch1-url="Running-a-Server" %}
+
+# Running an Unregistered Server
+
+It is highly recommended to test your Server by registering it on one of the built-in Directories **first**. This will help you tackle general problems before you try unregistered mode.
+
+## Setting up a Server behind a home router
+
+If you set up your server at home, you will probably need to change some settings in your router/firewall:
+
+### Port forwarding
+
+Normally, people from outside your home network cannot see things inside it. To let external Jamulus clients connect to your server, you need to set up port forwarding in your router's settings. The exact setup differs for every router. For help, see your router's documentation or [portforward.com](https://portforward.com).
+
+**Note:** The default port for Jamulus is **UDP** (not TCP) port **22124**. You will usually forward the port **22124** from outside your network to the port **22124** of the computer running the Server.
+
+**Note:** Your home router might change the IP address of the computer you're running your Server on. Depending on your router, you might need to give this computer a static IP address (often under DHCP settings of your router).
```

Comment:
This note is in the port forwarding section. Should it be?

Reply:
Oh might be a mistake I'll have a look.